internal/deviations/deviations.go

```diff
@@ -490,6 +490,29 @@ func StorageComponentUnsupported(_ *ondatra.DUTDevice) bool {
 	return *storageComponentUnsupported
 }
 
+// PlatformPowerEnableWait returns timer value to wait while platform is powered on.
+// Default value is 0.
+func PlatformPowerEnableWait(dut *ondatra.DUTDevice) int64 {
+	ppewait := lookupDUTDeviations(dut).GetPlatformPowerEnableWait()
+	var min_ppwewait int64 = 0
+	if ppewait > min_ppwewait {
```

Comment:
Curious, default value returned by `GetPlatformPowerEnableWait()` is 0.

If the default value of the proto is 0, then would it make sense to directly return `lookupDUTDeviations(dut).GetPlatformPowerEnableWait()` ?